main.py

```diff
@@ -0,0 +1,34 @@
+import nbp
+import db
+import datetime
+
+
+def nbp_rates_test():
+    dates = []
+    rateseur = []
+    ratesusd = []
+    resp = nbp.get_avg_rates('EUR', 120)
+    print("Pobrano kursy EUR z ostatnich 120 dni:")
+    for item in resp:
+        dates.append(datetime.datetime.strptime(item['effectiveDate'], "%Y-%m-%d"))
+        rateseur.append(item['mid'])
+        print('{} {}'.format(item['effectiveDate'], item['mid']))
+    resp = nbp.get_avg_rates('USD', 120)
```

Comment:
to 120 jeśli już to do stałej. Poza tym warto przemyśleć czy chcemy tak się ograniczać do 120, bo to znacząco utrudniłoby dalszy rozwój tych funkcji pobierających- np. by przystosować do pracy z roznymi przedzialami dat